lib/gateway.js

```diff
@@ -281,6 +289,47 @@ async function buildGateway (gatewayOpts, app) {
     schema,
     serviceMap,
     entityResolversFactory: factory,
+    pollingInterval: gatewayOpts.pollingInterval,
+    async refresh () {
+      if (this._serviceSDLs === undefined) {
+        this._serviceSDLs = serviceSDLs.join('')
+      }
+
+      await allSettled(
+        Object.values(serviceMap).map((service) =>
+          service.refresh().catch((err) => {
+            errorHandler(err, service)
+          })
+        )
+      ).then((result) => {
```

Comment:
nit: can you remove `then()` and use another `await`?

Reply:
Yea, I will replace `.then()` with variable. I wanted to avoid extra variables there.